lib/matplotlib/axes/_axes.py

```diff
@@ -3427,13 +3427,17 @@ def extract_err(name, err, data, lolims, uplims):
                 the note in the main docstring about this parameter's name.
             """
             try:
-                low, high = np.broadcast_to(err, (2, len(data)))
```

Comment:
Sorry I looked at this so late. A simpler solution here is to add `subok=True`...

Reply:
Just today, I decided to look at how matplotlib does this, because I need a similar interface (accept a scalar and 1 d array or a 2 d array - just like err here. I guess that's a rare case that that happens, but should I open a PR to simplify this back to `np.boradcast_to` with `subok`?